### ISSUE TITLE
Allow to syndicate to reply/like/repost url no matter what

### ISSUE THREAD

**Issue by swentel** [swentel/indigenous-android#19]
currently, it's only possible to syndicate when a url has come in from the share intents, open this up

**Comment by swentel** [comment]
This should actually just go away, when there's a url, this should happen automatically server side.

**Comment by dshanske** [comment]
What about syndicating a note?

**Comment by swentel** [comment]
A note currently has no url field in the UI, is that something that should be possible ?

**Comment by dshanske** [comment]
I want to write a note on my phone and post it on my website and have my site syndicate it to Twitter.

**Comment by dshanske** [comment]
So, I check the syndication target and it creates the post and syndicates it.

**Comment by dshanske** [comment]
https://www.w3.org/TR/micropub/#examples-of-creating-objects

**Comment by dshanske** [comment]
mp-syndicate-to = https://myfavoritesocialnetwork.example/aaronpk, https://archive.org/, etc.

    This property is giving a command to the Micropub endpoint, rather than just creating data, so it uses the mp- prefix.

**Comment by dshanske** [comment]
The UID sent to mp-syndicate-to triggers a POSSE instruction

**Comment by swentel** [comment]
Oh, that's still there for every post type :)

What I had at some point was a checkbox for the reply/like/bookmark/repost type where I added an additional checkbox which said something like 'To reply-url', in addition to existing syndication targets like twitter or facebook. When you toggled that one, I also added the url in the mp-syndicate-to array as I thought that was the way to notify a server that a webmention should be send to a 'custom' url. However, I misunderstood that part a bit  (although somehow, I think it's a good idea though to  have a property like say mp-sendmention-to-url, but that's a totally different discussion).
